includes/Modules/TagManager.php

```diff
@@ -459,18 +492,28 @@ protected function parse_data_response( $method, $datapoint, $response ) {
 					} else {
 						$account_id = $response['accounts'][0]->getAccountId();
 					}
+
 					$containers = $this->get_data( 'list-containers', array( 'accountId' => $account_id ) );
+
 					if ( is_wp_error( $containers ) ) {
 						return $response;
 					}
+
 					return array_merge( $response, $containers );
 				case 'list-containers':
-					$response = array(
+					$account_id = $this->_containers_account_id;
```

Comment:
Can you wrap this in a check whether the property is set first? And after reading the property, unset it back to `null`.

Reply:
sure, I was assuming accountId param is always present when requesting list-containers here https://github.com/google/site-kit-wp/pull/335/files#diff-b2484d51f9cd2114eca74b872d92b73eR359 as it would throw error if not but better to add extra verification. thanks!